pkg/trace/stats/statsraw.go

```diff
@@ -8,26 +8,68 @@ package stats
 import (
 	"strings"
 
-	"github.com/DataDog/datadog-agent/pkg/trace/stats/quantile"
+	"github.com/DataDog/datadog-agent/pkg/trace/pb"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+
+	"github.com/DataDog/sketches-go/ddsketch"
+	"github.com/golang/protobuf/proto"
+)
+
+const (
+	// relativeAccuracy is the value accuracy we have on the percentiles. For example, we can
+	// say that p99 is 100ms +- 1ms
+	relativeAccuracy = 0.01
+	// maxNumBins is the maximum number of bins of the ddSketch we use to store percentiles.
+	// It can affect relative accuracy, but in practice, 2048 bins is enough to have 1% relative accuracy from
+	// 80 micro second to 1 year: http://www.vldb.org/pvldb/vol12/p2195-masson.pdf
+	maxNumBins = 2048
 )
 
 // Most "algorithm" stuff here is tested with stats_test.go as what is important
 // is that the final data, the one with send after a call to Export(), is correct.
 
 type groupedStats struct {
-	topLevel float64
+	hits            float64
```

Comment:
What's the reason why the counts are float64 here but uint64 in the proto?

Reply:
No good reason, I just didn't change it. Now it makes a lot more sense indeed. Updated.